app/actions/websocket/websocket.test.js

```diff
@@ -119,15 +120,34 @@ describe('Actions.Websocket doReconnect', () => {
                 myPreferences: {},
             },
             posts: {
-                posts: {},
-                postsInChannel: {},
+                posts: {
+                    [post1.id]: post1,
+                },
+                postsInChannel: {
+                    [channel1.id]: [
+                        {
+                            recent: true,
+                            order: [post1.id],
+                        },
+                    ],
+                },
+            },
+            apps: {
+                pluginEnabled: true,
```

Comment:
@enahum Adding these to the test's state is necessary to make the `handleRefreshBindings` call occur. Otherwise `getPostIdsInChannel` will return null and cause other selectors to fail.

The `BATCH_WS_RECONNECT` is now occurring in two tests below, when the tests used to assert that it didn't occur. I think this is because it was incorrectly assumed that it _shouldn't_ happen, and the only reason it wasn't happening before is because the test wasn't set up properly.